WebNetSample/Views/Products/Edit.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@using WebNetSample.Entity.Concrete;
+@model Product
+
+<p>
+    <h3 class="text-warning">You are editing <span class="text-danger">@Model.Name</span> product</h3>
+</p>
+
+@using (Html.BeginForm("Update", "Products", FormMethod.Post))
+{
+    <table cellpadding="0" cellspacing="0">
+        <tr>
+            <th colspan="2" align="center">Product Details</th>
+        </tr>
+        <tr>
+            <td>Id: </td>
+            <td>
+                @Html.TextBoxFor(m => m.Id, new { @readonly = "readonly", @Value = @Model.Id })
+            </td>
+        </tr>
+        <tr>
+            <td>Name: </td>
+            <td>
+                @Html.TextBoxFor(m => m.Name, new { @Value = @Model.Name })
+            </td>
+        </tr>
+        <tr>
+            <td>Category: </td>
+            <td>
+                @Html.DropDownListFor(m => m.CategoryId, new List<SelectListItem>
```

Comment:
Since this row is too long it is better to wrap it into column:
```suggestion
                @Html.DropDownListFor(
                           m => m.CategoryId,
                           new List<SelectListItem> { new SelectListItem{Text="Toys", Value="cc1fd238-f590-483e-ab5c-15006f73eb92"}}, 
                           "Please select a category")
```

BTW, you can simplify 37 row like (C# 10 feature):
```suggestion
                new List<SelectListItem> { new() { Text="Toys", Value="cc1fd238-f590-483e-ab5c-15006f73eb92" }}, 
```